lisa/tools/tpm2.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Dict, List, Optional, Sequence, Union
+
+from lisa.executable import Tool
+from lisa.operating_system import CBLMariner
+from lisa.util import LisaException
+
+
+class Tpm2(Tool):
+    @property
+    def command(self) -> str:
+        return "tpm2"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def _install(self) -> bool:
+        if isinstance(self.node.os, CBLMariner):
+            self.node.os.install_packages("tpm2-tools")
+        else:
+            raise LisaException(
+                f"tool {self.command} can't be installed in distro {self.node.os.name}."
+            )
+        return self._check_exists()
+
+    def pcrread(
+        self, alg: str = "sha256", pcrs: Optional[Union[int, Sequence[int]]] = None
```

Comment:
Remove the unused argument `alg`.

Reply:
It is used here
```python3
cmd = f"pcrread {alg}:{pcrs_arg}"
```

Reply:
I mean it's not assigned value when calling the method. So it should be removed, until it's really used when calling the method.

Reply:
ah okay. Will do